htdocs/install/mysql/migration/18.0.0-19.0.0.sql

```diff
@@ -218,3 +218,5 @@ ALTER TABLE llx_bom_bomline ADD COLUMN tms timestamp DEFAULT CURRENT_TIMESTAMP O
 UPDATE llx_c_type_contact SET element = 'stocktransfer' WHERE element = 'StockTransfer';
 
 UPDATE llx_c_units SET scale = 1 WHERE code = 'S';
+
+UPDATE llx_c_tva SET taux = 22, note = 'Νήσων υπερμειωμένος Φ.Π.Α.' WHERE fk_pays = 102 AND taux = 6;
```

Comment:
In llx_c_tva, you replace the vat rate 16 into 3
But into the upgrade you update the 22 into 6
What is the good one ?

Reply:
No excuse, stupid mistake...fixed